types/inflation/index.d.ts

```diff
@@ -18,10 +18,20 @@ export = inflate;
 declare function inflate(req: Readable, options?: inflate.Options): Readable;
 
 declare namespace inflate {
-    interface Options extends ZlibOptions {
```

Comment:
`interface Options extends ZlibOptionsBase {` is not enough?

Reply:
Practically speaking? Perhaps. I believe the `info` option only does something when used with deflate, deflateRaw, gzip, gunzip, inflate, inflateRaw or unzip but it would deviate from the official Node.JS documentation (it doesn't clarify whether the `info` option is ignored for the classes or not) and the TypeScript definitions so that is the reason why I added both flavors.

Reply:
By second thought, I reverted changes to the `ZlibOptions`  interface, my proposed change of it being an union type of my two new introduced interfaces 'ZlibOptionsWithoutInfo` and `ZlibOptionsWithInfo` has hereby made undone, making changes to the `@types/inflation` package no longer necessary and thus have been reverted as well so this PR no longer affects it.